docs/modules/ROOT/pages/manage/relayers.adoc

```diff
@@ -110,13 +110,13 @@ const client = new Defender({
   relayerApiSecret: 'YOUR_API_SECRET'
 });
 
-const tx = await client.relayerSigner.sendTransaction({
+const tx = await client.relaySigner.sendTransaction({
   to, value, data, gasLimit, speed: 'fast'
 });
-
-const mined = await tx.wait();
 ----
 
+NOTE:  There are a few options available for checking transaction status. You can set up a notification webhook channel for push-based notifications, or use a polling-based approach. In the polling-based method, you can either call the *getTransaction* method by passing the transactionId, or use the *getTransactionByNonce* method by passing the nonce.
```

Comment:
Should we add that this is currently not supported for RG?

Reply:
Not sure, maybe we should create section specific to groups and mention differences. Let's leave it out for now.